README.adoc

```diff
@@ -171,6 +175,29 @@ nix = {
 NOTE: If you are a https://nixos.org/nix/manual/#ssec-multi-user[trusted user] you may add the
 `nix.conf` lines to `~/.config/nix/nix.conf` instead.
 
+
+==== Cache setup via `.envrc`
+Alternatively you may want to configure the cache only for the plutus project. If you use
+lorri and/or direnv you can do so by adding the following to the `.envrc` at the project root:
+
+----
+# Create a local nix config file
+NIX_CONF_DIR=$(pwd)/.config/nix
+mkdir -p $NIX_CONF_DIR
+
+# Store the iohk specific cache config
```

Comment:
We could just tell them to write this file to `.config/nix/nix.conf` in the instructions rather than redoing it every time we hit `.envrc`. Or we could actually check that in somewhere, which would have the example of being easy to update if we do ever change it!

Reply:
Well the whole idea with this is for this to be a "zero manual setup" solution so writing instructions on where to put something somewhat defeats the purpose. 

Since we are basically just piping a few lines into a file i don't care that much about doing it repeatedly. I don't notice any kind of delay through it.

Reply:
Why not just check in `.config/nix/nix.conf` into our repo?

Reply:
Yep, that would work of course!